desktop/appconfig.h

```diff
@@ -0,0 +1,26 @@
+#ifndef APPCONFIG_H
+#define APPCONFIG_H
+
+#include <QString>
+
+// This class is intended to store app configuration
+// modifiable from JS side
+// Currently, only logging-related settings are here
+// that are used by react-native-desktop-config module
+class AppConfig {
+public:
+
+  static AppConfig* inst();
+
+  bool getLoggingEnabled() const;
+  void setLoggingEnabled(bool enable);
+
+private:
+  AppConfig();
+
+  bool loggingEnabled;
```

Comment:
It would be nice to have default value for `loggingEnabled` set.

Reply:
Done.